Assets/Scripts/Samplers/ScatterMap.cs

```diff
@@ -35,7 +34,8 @@ public class ScatterMap : MonoBehaviour
         private Coroutine scatterCoroutine;
         private Queue<IEnumerator> coroutineQueue = new Queue<IEnumerator>(); //queue to ensure the camera only processes one request at a time
 
-        public ScatterSettingsPropertySection propertyPanelPrefab; //todo: find a better way to reference this.
+        public ScatterSettingsPropertySection scatterProptertiesPrefab; //todo: find a better way to reference this.
```

Comment:
Deze variabele is hernoemd, moet ik er een [FormerlySerializedAs] by laten staan? ik heb de referentie handmatig opnieuw geassigned en er was maar 1 instance van dit script